cms/templates/asset_index.html

```diff
@@ -61,7 +58,9 @@ <h3 class="page-header">
     <div class="content">
         <div class="content-primary">
             % if waffle_flag_enabled:
-                <div id="root"></div>
+                <%static:webpack entry="AssetsPage">
+                    <div id="root"></div>
+                </%static:webpack>
```

Comment:
This is super borked ATM, not appearing at all. I think my mako syntax is wrong?